docs/ghz/README.md

```diff
@@ -41,7 +38,20 @@ Example `config.json`:
 }
 ```
 
-For the complete list of configuration parameter, please check [ghz documentation][ghz params].
+### Providing protobuf schema
+
+While `ghz` accepts `.proto` files to not depend on server reflection, Kangal currently only supports `.protoset` files.
+
+To do so, use the `testData` form field to provide the `.protoset` file and add the following key to your JSON configuration file:
+
+```json
+{
+  "protoset": "/data/testdata.protoset",
+  ...
+}
+```
+
+For information about how to create `.protoset` files and the complete list of configuration parameter, please check [ghz documentation][ghz params].
```

Comment:
I think we should explain how to generate a `protoset` file; either by linking to the specific documentation or writing the command in the readme.